README.md

```diff
@@ -111,11 +111,17 @@ Compile the project:
 These are the command-line arguments that can be provided to the
 `erlang_ls` escript:
 
-| Argument                | Description                                                                                     |
-|-------------------------|-------------------------------------------------------------------------------------------------|
-| --transport tcp / stdio | Specifies the transport the server will use for the connection with the client (default: `tcp`) |
-| --port PORT             | Used when the transport is `tcp` (default: `10000`)                                             |
-| --log-dir DIR           | Directory where logs will be written. When not provided no logs are generated.                  |
+``` shell
+Usage: Erlang LS [-v] [-t [<transport>]] [-p [<port>]] [-d [<log_dir>]]
+                 [-l [<log_level>]] [<port_old>]
+
+  -v, --version    Print the current version of Erlang LS
+  -t, --transport  Specifies the transport the server will use for the
+                   connection with the client. [default: tcp]
+  -p, --port       Used when the transport is tcp. [default: 10000]
+  -d, --log-dir    Directory where logs will be written. [default: <user_log>/erlang_ls/log]
```

Comment:
Not sure about the default value. Is there really a `log` as the last section in the path?

Reply:
It is what getopt and erl are both telling me :) 

```erlang
Eshell V10.3.5.7  (abort with ^G)
1> filename:basedir(user_log, "erlang_ls").
"/home/onnovos/.cache/erlang_ls/log"
```

```shell
 [OTP: 21.3.8.11][iex: 1.9.0] src/github/erlang_ls git:(476-use-getopt-for-cli-arg-parsing)
erlang_ls -h
Usage: Erlang LS [-v] [-t [<transport>]] [-p [<port>]] [-d [<log_dir>]]
                 [-l [<log_level>]] [<port_old>]

  -v, --version    Print the current version of Erlang LS
  -t, --transport  Specifies the transport the server will use for the
                   connection with the client. [default: tcp]
  -p, --port       Used when the transport is tcp. [default: 10000]
  -d, --log-dir    Directory where logs will be written. [default:
                   /home/onnovos/.cache/erlang_ls/log]
  -l, --log-level  The log level that should be used. [default: info]
  <port_old>       Port provided as integer for backwards compatibility
                   reasons
```

Reply:
I've taken the output from `erlang_ls -h` and "masked" out my path with `<user_log>` which seemed to make most sense as a "variable". If you have another suggestion I'm happy to change it :)

Reply:
The value is platform dependent. 

This is in Mac OS X for example:
```
Eshell V10.3.5.6  (abort with ^G)
1> filename:basedir(user_log, "erlang_ls").
"/Users/juan.facorro/Library/Logs/erlang_ls"
2>
```
We could show the actual call to `filename:basedir(user_log, "erlang_ls")` or mention that it's platform dependent and to check the documentation.
```
-d, --log-dir    Directory where logs will be written. [default:
                 filename:basedir(user_log, "erlang_ls")]
```
or
```
-d, --log-dir    Directory where logs will be written. [default:
                 platform dependent, please check the documentation]
```

Reply:
I'll change it to the first example you give. getopts automatically expands this so didn't realize that also the suffix is apparently different (missing log). 

Push coming up 👍

Reply:
Thanks @jfacorro for the clarification and @onno-vos-dev for the fix!